### PR TITLE
update "Steps to Join Rocketchat" location to non-archived version

### PR DESCRIPTION
**Pull request by caggles**
The devhub was pointing to an archived repo instead of the one where reggie is actively being developed!

### CODE REVIEW

**Review by patricksimonian — APPROVED** [review]

**Comment by patricksimonian** [comment]
@deploy-tron deploy web to dev

**Comment by patricksimonian** [comment]
@deploy-tron deploy web to dev

**Comment by patricksimonian** [comment]
@deploy-tron deploy web to dev

File: web/topicRegistry/getting-started-on-the-devops-platform.json [hunk 1]
```diff
@@ -119,6 +119,17 @@
             "docs/OnboardingOutcomes.md"
           ]
         }
+      },
+      {
+        "sourceType": "github",
+        "sourceProperties": {
+          "url": "https://github.com/bcgov/networkpolicy-migration-workshop",
+          "owner": "bcgov",
+          "repo": "networkpolicy-migration-workshop",
```
Comment (patricksimonian) [inline web/topicRegistry/getting-started-on-the-devops-platform.json:128]:
```suggestion
          "repo": "networkpolicy-migration-workshop",
          "ref": "main",
```

**Comment by patricksimonian** [comment]
@deploy-tron deploy web to dev

**Comment by patricksimonian** [comment]
@deploy-tron deploy web to dev

**Comment by patricksimonian** [comment]
@deploy-tron deploy web to dev

**Comment by patricksimonian** [comment]
@deploy-tron deploy web to dev

**Comment by patricksimonian** [comment]
@deploy-tron deploy web to dev

**Comment by patricksimonian** [comment]
@deploy-tron deploy web to test

**Comment by AndrewJun97** [comment]
@deploy-tron deploy test to prod

**Comment by patricksimonian** [comment]
@deploy-tron deploy web to prod

File: web/topicRegistry/community-and-events.json [hunk 2]
```diff
@@ -153,4 +153,4 @@
       }
     ]
   }
-}
\ No newline at end of file
+}
```
Comment (patricksimonian) [inline web/topicRegistry/community-and-events.json:156]:
```suggestion
}

```

**Comment by AndrewJun97** [comment]
@deploy-tron deploy web to dev

**Comment by AndrewJun97** [comment]
@deploy-tron deploy web to test

**Comment by AndrewJun97** [comment]
@deploy-tron deploy web to dev

**Comment by AndrewJun97** [comment]
@deploy-tron deploy web to test

**Comment by AndrewJun97** [comment]
@deploy-tron deploy web to prod

**Pull request merged**